app/views/api/v1/transactions/create.json.jbuilder

```diff
@@ -0,0 +1,14 @@
+json.transaction @transaction, partial: 'transaction', as: :transaction
+
+json.transaction do
+  json.category @transaction.category, :id, :name, :type
```

Comment:
@yunixon move it to `categories/_short_category.json.jbuilder` partial. And render it in `_transaction` partial.